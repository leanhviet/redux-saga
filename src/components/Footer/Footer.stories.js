// Libs
import React from 'react'
import { storiesOf } from '@storybook/react'

// Components
import Footer from './index'

storiesOf('Footer', module).add('Default', () => <Footer />)
