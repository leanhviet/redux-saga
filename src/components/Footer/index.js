import React from 'react'

const Footer = () => (
  <footer className="footer">
    <h4 className="footer__copyright">Copyright &copy; 2019 Youtube Inc.</h4>
  </footer>
)

export default Footer
