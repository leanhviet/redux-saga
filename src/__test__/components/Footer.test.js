// Libs
import React from 'react'
import { shallowToJson } from 'enzyme-to-json'
import { shallow } from 'enzyme'

// Component
import Footer from '../../components/Footer'

describe('Footer component', () => {
  // Snapshot
  test('should render correctly Footer component default props', () => {
    const component = shallow(<Footer />)

    expect(shallowToJson(component)).toMatchSnapshot()
  })
})
