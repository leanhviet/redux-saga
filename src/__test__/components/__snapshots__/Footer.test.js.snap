// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly Footer component default props 1`] = `
<footer
  className="footer"
>
  <h4
    className="footer__copyright"
  >
    Copyright Â© 2019 Youtube Inc.
  </h4>
</footer>
`;
